frontend/cypress/support/index.js

```diff
@@ -13,6 +13,14 @@
 // https://on.cypress.io/configuration
 // ***********************************************************
 
+Cypress.Commands.overwrite("visit", (originalVisit, url, options) => {
```

Comment:
On the way added consistent locale in e2e tests so you can test titles locally even if your locale is not `en`

Reply:
Good improvement 👍